crates/op-rbuilder/src/builders/flashblocks/builder_tx.rs

```diff
@@ -1,28 +1,96 @@
 use alloy_consensus::TxEip1559;
-use alloy_eips::{Encodable2718, eip7623::TOTAL_COST_FLOOR_PER_TOKEN};
-use alloy_evm::Database;
-use alloy_primitives::{Address, TxKind};
+use alloy_eips::Encodable2718;
+use alloy_evm::{Database, Evm};
+use alloy_op_evm::OpEvm;
+use alloy_primitives::{Address, Bytes, TxKind, U256};
+use alloy_sol_types::{SolCall, SolError, sol};
 use core::fmt::Debug;
 use op_alloy_consensus::OpTypedTransaction;
+use op_revm::OpHaltReason;
+use reth_evm::{ConfigureEvm, precompiles::PrecompilesMap};
 use reth_optimism_primitives::OpTransactionSigned;
 use reth_primitives::Recovered;
 use reth_provider::StateProvider;
-use reth_revm::State;
+use reth_revm::{State, database::StateProviderDatabase};
+use revm::{
+    context::result::{ExecutionResult, ResultAndState},
+    inspector::NoOpInspector,
+};
 
 use crate::{
     builders::{
         BuilderTransactionCtx, BuilderTransactionError, BuilderTransactions,
-        context::OpPayloadBuilderCtx, flashblocks::payload::FlashblocksExtraCtx,
+        builder_tx::{BuilderTxBase, get_nonce},
+        context::OpPayloadBuilderCtx,
+        flashblocks::payload::FlashblocksExtraCtx,
     },
     flashtestations::service::FlashtestationsBuilderTx,
     primitives::reth::ExecutionInfo,
     tx_signer::Signer,
 };
 
+sol!(
```

Comment:
do we foresee this code changing? it would be good to have a link/specific commit referenced for this

Reply:
its subject to change depending on whether the flashblocks contract will rely on the flashtestations contract for auth. but I can link the flashblock contract repo